src/System.Xml.ReaderWriter/src/System/Xml/AsyncHelper.cs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Xml
+{
+    internal static class AsyncHelper
+    {
+        public static readonly Task DoneTask = Task.FromResult(true);
```

Comment:
I'm not sure there is a point of that as cleaning up is usually never-ending story. Usually during port PR we review only stuff like: .csproj files, packages.config etc.